storybookjs/storybook
# Change viewport styling to more closely match v4

Issue: #5965

## What I did

A regression in #7385 re-added the subjectively ugly black border that had been removed in #5979 to address #5965. This PR reverts the border to its v4-like thinness.

## How to test

- Is this testable with Jest or Chromatic screenshots? this addon doesn't seem to have tests
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
